src/app/home/home.component.html

```diff
@@ -68,7 +68,39 @@ <h2 *ngIf="type.value === 'f'">{{ weather.current.temp_f }} F</h2>
       </mat-tab>
       <mat-tab 
         label="Next days">
-        <!-- NEXT 5 DAYS -->
+        <mat-card *ngIf="weathers && !isLoading">
+          <mat-grid-list cols="5" rowHeight="1:1">
+            <mat-grid-tile *ngFor="let current of weathers.forecast.forecastday">
+              <mat-card-content>
+                  <p>
+                    {{ current.day.condition.text }}
+                  </p>
+                  <img
+                    [src]="current.day.condition.icon"
+                    alt="Cloudiefy icon">
+                  <h2 *ngIf="type.value === 'c'">{{ current.day.avgtemp_c }} C</h2>
+                  <h2 *ngIf="type.value === 'f'">{{ current.day.avgtemp_f }} F</h2>
+                  <p>
+                    Humidity:
+                    <strong>
+                      {{ current.day.avghumidity }} %
```

Comment:
Is there a way to extract `day` object to avoid repeating `current.day`? Same happens before with `weather.current`

Reply:
It was changed. Thanks